apps/frontend/src/components/generic/ApexLineChart.vue

```diff
@@ -39,15 +38,18 @@ export interface SeriesItem {
 export default class ApexLineChart extends Vue {
   @Prop({required: true, type: Array}) readonly categories!: Category<string>[];
   @Prop({required: true, type: Array}) readonly series!: number[];
-  @Prop({type: Number}) readonly upper_range!: number; //upper bound of y axis
-  @Prop({type: Boolean}) readonly sev_chart!: boolean; //identifies chart as severity chart
+  @Prop({type: Number}) readonly upperRange!: number; //upper bound of y axis
+  @Prop({type: Boolean}) readonly sevChart!: boolean; //identifies chart as severity chart
   @Prop({type: String}) readonly title!: string;
-  @Prop({type: String}) readonly y_title!: string;
+  @Prop({type: String}) readonly yTitle!: string;
 
-  chart_id: string = `line_chart_${next_id}`;
+  chartId= `line_chart_${next_id}`;
```

Comment:
This line isn't actually used anywhere and is the only location that `next_id` is used in, allowing both this and `next_id` and `idCounter` to be removed.